### DISCUSSION TITLE
Miscount presses in key overlay during replay.

### DISCUSSION THREAD

**Discussion by Semro** [ppy/osu#15026] [Q&A] [answered]
Rewinding to the beginning of the map doesn't descend press count as it would be with Autoplay mod.
The replay was made in the same osu!lazer version.

beatmap: https://osu.ppy.sh/beatmapsets/1207609#osu/2514776
replay: [replay](https://github.com/ppy/osu/files/7317284/Semro.playing.Seth.Everman.-.how.to.create.the.weeknd.s._blinding.lights_.browiec.insane.zip)

osu!lazer 2021.1006.1
[performance.log](https://github.com/ppy/osu/files/7317275/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7317276/runtime.log)

https://user-images.githubusercontent.com/3332159/136692374-e866c656-eb4f-4c44-9544-3577d7dfabee.mp4

**Comment by smoogipoo** [comment, marked as answer]
Issue: https://github.com/ppy/osu/issues/15027
